service/service.go

```diff
@@ -120,7 +120,7 @@ func New(tarball io.Reader, env map[string]string, options ...Option) (*Service,
 	if err != nil {
 		return nil, err
 	}
-	s.Hash = hex.EncodeToString(hash)
+	s.Hash = base58.Encode(hash)
```

Comment:
I'm not sure here if the idea was to store the bytes in the raw data and just have the representation for docker and for the grpc api.
I'm fine with that, just want to have some clarification about that

Reply:
Storing has as bytes is a different case and should be done in different PR.